roles/jitsi-meet/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
in der ansible galaxy gibt es einige jitsi rollen die ganz gut aussehen - warum ne eigene statt nem submodule einer existierenden?